### DISCUSSION TITLE
漢化修正/追加建議

### DISCUSSION THREAD

**Discussion by Liteya** [GpointChen/FFXIVChnTextPatch-GP#339] [General]
resource/rawexd/quest/046/BanOmi102_04608.csv
原  
24,TEXT_BANOMI102_04608_TODO_00	在「說話」頻道下輸入「咖啡時間」<hex:02100103>呼喚正在至福樂土進行農耕的卡列連人們
68,TEXT_BANOMI102_04608_SAYTODO_000_050,咖啡時間

修改
24,TEXT_BANOMI102_04608_TODO_00,在「說話」頻道下輸入「咖啡時間」（coffee break）<hex:02100103>呼喚正在至福樂土進行農耕的卡列連人們
68,TEXT_BANOMI102_04608_SAYTODO_000_050,coffee break

還有什麼要調整的嘛?照慣例say 的部份是改英文的沒錯吧? 其實我不太懂要怎麼調整..
------------------
還有一個在 開拓島的部分 
resource/rawexd/custom/007/CtsMjiVillageDevelopment_00782.csv 

有追加了幾行文字 或者 沒翻譯的部分留白是嘛?

-------------------------------

8,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_Q1_110_006,
9,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_Q1_110_010,現在先算了
10,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_120_000,你身上的謝爾達萊青船幣好像不夠呢……<hex:02100103>等你攢夠了再來找我吧！
11,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_120_005,明白，等你什麼時候想要開墾土地，<hex:02100103>再來找我吧！
12,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU2_120_002,拓開這片森林之後，就能擴大據點了。<hex:02100103>在此之前，你還需要收集一些開拓用具和我們的夥伴。
13,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU3_120_003,推平這些土石之後，就能擴大據點了。<hex:02100103>在此之前，你還需要收集一些開拓用具和我們的夥伴。
14,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU4_120_004,去除這些荊棘之後，就能擴大據點了。<hex:02100103>在此之前，你還需要收集一些開拓用具和我們的夥伴。
15,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU5_120_005,碎開這塊巨岩之後，就能擴大據點了。<hex:02100103>在此之前，你還需要收集一些開拓用具和我們的夥伴。
16,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU5_120_006,ここの山肌を大きく削ったら、拠点を広げられるかも……？<hex:02100103>在此之前，你還需要收集一些開拓用具和我們的夥伴。
17,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_200_000,你好呀！歡迎你來玩！<hex:02100103>希望你在島上玩得開心！
18,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_021_000,我聽貓耳小員說了！<hex:02100103>要開始準備整理新的土地嗎？
19,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_021_000,要開放新的土地嗎？
20,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_Q1_021_000,趕緊開放吧！
21,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_Q1_021_005,現在先算了
22,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_021_005,好！我這就開始整理土地！
23,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_021_010,明白，我會一直在這裡等你的。<hex:02100103>你有需要時再來找我吧。
24,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_022_000,呼……雖然花了一些功夫，但是土地擴大完成啦！<hex:02100103>如果你希望進一步擴大據點，就需要使用謝爾達萊青船幣<hex:02100103>讓其他的魔法人偶來幫忙！
25,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_022_005,建造開拓工房用的土地就準備好啦！<hex:02100103>這兩片土地，你可以挑喜歡的一邊建造開拓工房。<hex:02100103>先來造一座建築物試試吧！
26,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_022_010,建造開拓工房需要石灰岩，<hex:02100103>你用現有的道具是沒法採集石灰岩的。<hex:02100103>我們來做一把開拓用石錘吧！
27,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_022_015,只要有了開拓用石錘，<hex:02100103>就能收集到開拓工房所需的全部建材啦！<hex:02100103>你可以使用建築管理面板查看需要什麼建材。
28,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_022_020,對了對了！<hex:02100103>因為你開拓了土地，擴大了開拓據點，<hex:02100103>所以現在這裡可以容納寵物們在這裡玩耍啦！
29,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_022_000,學會開拓用石錘的開拓製作配方了！<hex:02100103>使用開拓用石錘可用於擊碎岩石獲取石材。
30,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_022_005,可以放養寵物了！<hex:02100103>在開拓管理面板切換至管理寵物，<hex:02100103>即可將你擁有的寵物放養至開拓據點內活動。
31,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_022_010,各功能的詳細介紹，<hex:02100103>請參考開拓管理面板的「查看說明」項目。
32,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_022_015,開拓了新的土地！<hex:02100103>今後，進一步地開拓新土地時，需要支付謝爾達萊青船幣，<hex:02100103>作為魔法人偶的運轉費用。
33,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_022_020,「無人島庭具投影」が開放されました！<hex:02100103>開拓管理ボードの「無人島庭具管理」から、<hex:02100103>所持している庭具の幻影を、開拓拠点内に投影できます。
34,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_023_000,要是不知道怎麼做開拓用石錘的話，<hex:02100103>可以看看開拓製作中的配方哦！
35,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_024_000,你可以用管理板查看開拓工房所需的素材！
36,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_025_000,終於！開拓工房建成啦！<hex:02100103>有了開拓工房，我的魔法人偶同伴們<hex:02100103>就可以在這裡用收集到的開拓素材製作海島產品啦！
37,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_025_005,我們做出來的海島產品，<hex:02100103>可以通過群島貿易賣出去賺取謝爾達萊青船幣！<hex:02100103>做更多的海島產品！賺大錢！
38,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_KAITAKU1_025_010,啊！對了！<hex:02100103>這一次是情況特殊，特地趕工造好了房子，<hex:02100103>下一次建造新建築的時候，每一棟都要花上一段時間才能完成的。
39,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_025_000,可以使用開拓工房了！<hex:02100103>與小島木屋附近的監工小員交談，<hex:02100103>即可指示魔法人偶們生產海島產品。
40,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_025_005,今後，建造開拓工房一類的建築物時，<hex:02100103>均需要等待一定的時間（地球時間）。
41,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_025_010,還有，除了小島木屋以外的建築物，<hex:02100103>均可以通過管理板上的搬遷選項移動到別處。
42,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_025_015,開拓等級的上限提升了！<hex:02100103>提升開拓等級即可學會新的開拓製作配方，<hex:02100103>或能夠建造新的建築物。
43,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_025_020,開拓工房では、設定したスケジュールに従い材料が自動消費されます。<hex:02100103>開拓管理ボードの「設定」より、<hex:02100103>消費予定の材料を使用する際の警告設定が行えます。
44,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_300_000,可以在開拓據點建造地標了！<hex:02100103>建造地標，可以提升開拓工房等設施的工作效率。
45,TEXT_CTSMJIVILLAGEDEVELOPMENT_00782_SYSTEM_300_005,（★未使用／削除予定★）
--------------------
其他的部分 在萬魔殿那邊的檔案好像也要調整 

resource/rawexd/custom/007/CtsOpn60Nemjiji_00732.csv

key,0,1
#,,
offset,0,4
int32,str,str
0,TEXT_CTSOPN60NEMJIJI_00732_CFC_TO_INSTANCE_CONTENT,<hex:02281CFF17>ContentFinderCondition<hex:E8020403>
1,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_000,<hex:022903EB0203>！感謝你願意協助我們調查。<hex:02100103>你報告中的那些過去的創造生物……<hex:02100103>呵呵呵……我已經等不及想要開始推測它們的生態了！
2,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_005,我想厄爾庇斯一定也有保管資料的設施，<hex:02100103>你先去那邊調查一下，再去萬魔殿吧。
3,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_010,你終於回來了！我真是等得望眼欲穿啊！
4,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_015,真不愧是富爾什諾大人……冷靜迅速地下達了指示。
5,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_020,我們遵從富爾什諾大人的指示，<hex:02100103>在這裡整理目前為止的調查結果。<hex:02100103>我對這一類的工作得心應手，放心交給我吧。
10,TEXT_CTSOPN60NEMJIJI_00732_Q1_000_000,要做什麼？
11,TEXT_CTSOPN60NEMJIJI_00732_A1_000_003,確認調查情況
12,TEXT_CTSOPN60NEMJIJI_00732_A1_000_004,取消
13,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_100,<hex:022903EB0203>，稍微耽誤你一點時間可以嗎？<hex:02100103>我想請你再詳細說說發生在過去世界的事。
14,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_101,水晶中發出的警告，還有萬魔殿的結局……<hex:02100103>總有一天，克勞狄恩老師會向學會報告<hex:02100103>此次事件的始末和相關現象。
15,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_102,為此，我希望能留下更加詳細的記錄。<hex:02100103>尤其是那些與你在邊境之獄交戰的危險對手！<hex:02100103>還請你務必具體講講！
16,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_103,尼姆吉吉！<hex:02100103>你要的資料我都已經找齊了，<hex:02100103>不過，那些真的都是解開水晶之謎所必需的資料嗎？
17,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_104,其中還有«水生生物的生態»和«各地的不死鳥傳說»等論文，<hex:02100103>都是和萬魔殿的創造生物有關的資料啊。
18,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_105,創造生物不是都已經被<hex:022903EB0203>打倒、重新封印起來了嗎？<hex:02100103>沒必要再深入調查了吧……
19,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_106,沒有的事。<hex:02100103>從現代研究者的視角調查存在於遠古時期的創造生物，<hex:02100103>這可是有著重大意義的。
20,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_107,當然……我個人也對此非常感興趣……
21,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_108,果然是出於你的個人興趣啊……<hex:02100103>不過與這回的事情也不是沒有關係，所以我連論文也替你拿來了，<hex:02100103>下次就拜託你自己收集了哦？
22,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_110,好了，我們言歸正傳。<hex:02100103>就請你講講那些十分危險的對手吧！
23,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_111,對了，變身後的埃里克特翁尼亞斯<hex:02100103>和化身為半神的獄卒長也拜託你一起講一下。<hex:02100103>這些都是很有趣……不，是令人興趣頗深的現象。
24,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_112,嗯嗯，謝謝你願意給我講這些。<hex:02100103>雖然很遺憾無法親眼去看看他們，<hex:02100103>但你的講述已經足以讓我做出詳細記錄了。
25,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_113,唔……<hex:02100103>乾脆給創造生物的生態加上簡單的推測，<hex:02100103>然後一起寫進記錄之中當作參考吧。
26,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_114,你知道嗎，現代的水生生物中，<hex:02100103>有的物種成年後能長到幼體的數十倍之大！
27,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_115,你遇到的魚尾海馬怪說不定也只是幼體，<hex:02100103>還遠遠沒有長大呢！
28,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_116,哎呀，沒想到你會這麼吃驚……<hex:02100103>在尚未確定創造生物是否會成長之前，<hex:02100103>這個推測確實有些過於跳躍了。
29,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_117,但是，想像力是無限的。<hex:02100103>在得到實證之前，一切皆有可能。<hex:02100103>啊，我的靈感正在源源不斷地湧現！
30,TEXT_CTSOPN60NEMJIJI_00732_Q1_100_100,要說什麼？
31,TEXT_CTSOPN60NEMJIJI_00732_A1_100_100,你要寫一份充滿妄想的記錄？
32,TEXT_CTSOPN60NEMJIJI_00732_A2_100_100,聽起來很有趣
33,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_120,嘿嘿嘿，推測的內容我會標註上「待檢驗」的！<hex:02100103>來吧，<hex:022903EB0203>，和我一起展開想像的翅膀！
34,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_140,<hex:022903EB0203>，可以再佔用你一點時間嗎？<hex:02100103>方便的話，可以給我講講你在煉淨之獄和絕地之獄的戰鬥詳情嗎？
35,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_141,雖然富爾什諾大人要求的事件梗概的報告已經提交上去了，<hex:02100103>但我想盡可能地留下更詳細的的記錄。
36,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_143,你啊……根本就是想滿足自己的好奇心吧？<hex:02100103>別以為我不知道，你前一陣還從所思大書院<hex:02100103>借閱了不少魔法生物和樹狀生物的論文。
37,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_144,哎呀，想要滿足自己好奇心這一點我不否認。<hex:02100103>但這也是為了克勞狄恩老師的研究嘛。<hex:02100103>我想<hex:022903EB0203>一定也能理解的。
38,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_145,嗯嗯……在煉淨之獄遇到的魔法生物<hex:02100103>居然和秘術師的寶石獸有共同點……<hex:02100103>或者說魔法生物這一概念也許就是從古代流傳下來的？
39,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_146,還有將樹狀生物與人融合……<hex:02100103>不論什麼樣的生物，都可以成為半神的材料嗎？<hex:02100103>真是有意思……
40,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_148,你果然只是想仔細研究那些創造生物和半神吧？
41,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_149,沒辦法，都是因為<hex:022903EB0203>的戰鬥太精彩了，<hex:02100103>我忍不住就興奮起來了！
42,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_150,不過聽<hex:022903EB0203>描述那些創造生物、<hex:02100103>半神的肉體特徵和戰鬥能力……<hex:02100103>感覺可以推測出很多東西！
43,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_100_150,比如說，赫淮斯托斯也許就在自己身上融入了<hex:02100103>「菲尼克司」所擁有的再生和不死的概念……<hex:02100103>呵呵呵，我腦子裡的妄想快要停不下來了！
44,TEXT_CTSOPN60NEMJIJI_00732_RUISSENAUD_000_151,我不阻止你妄想，<hex:02100103>但這份記錄也是要作為報告提交給富爾什諾大人的。<hex:02100103>你可別把那些不確定的推測也寫進去啊。
45,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_152,唉……要是克勞狄恩老師，一定會笑著同意的。
46,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_153,不過那是因為老師他也喜歡做這種推測。<hex:02100103>但把這種報告交給富爾什諾大人，<hex:02100103>只會得到一頓訓斥。
47,TEXT_CTSOPN60NEMJIJI_00732_Q1_100_200,要說什麼？
48,TEXT_CTSOPN60NEMJIJI_00732_A1_100_200,我覺得富爾什諾不會生氣的
49,TEXT_CTSOPN60NEMJIJI_00732_A2_100_200,惹富爾什諾生氣一定很恐怖
50,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_155,你說得對！富爾什諾大人眼界卓絕，<hex:02100103>他的心胸也一定像星海一樣寬廣！
51,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_160,連<hex:022903EB0203>也害怕富爾什諾大人啊……<hex:02100103>我明白了，我就只保留那些與報告內容有關的推測好了。
52,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_165,好了，接下來我就把從你這聽到的情報<hex:02100103>加上我自己的推測一起整理出一份記錄。
53,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_166,我會充分發揮想像力……然後等待推測被證實的那一天！<hex:02100103>來，<hex:022903EB0203>也來一起暢想吧！
73,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_400,根據克勞狄恩老師的要求，我們在向各機關報告時，<hex:02100103>只報告了最低限度的內容。
74,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_401,畢竟還不知道萬魔殿發生的事件會對現代造成什麼影響，<hex:02100103>所以也不方便做詳細報告。
75,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_402,我們和萬魔殿之間唯一的聯繫就是那個封印著記憶的水晶了……<hex:02100103>但是很可惜，你從中感知到的思念自那之後就再也沒出現過了。
76,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_403,雖然已經收集了靈魂水晶等封印著記憶的結晶進行比較，<hex:02100103>但目前還沒有得到任何成果……
77,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_404,要是我們也能和你一樣聽到裡面的聲音的話，<hex:02100103>肯定會更有利於調查吧……<hex:02100103>克勞狄恩老師也為此感到非常遺憾。
78,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_410,唉，克勞狄恩老師不在，研究也沒什麼進展。<hex:02100103>只能整理一下資料……
79,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_420,我很後悔，老師出發去搜索水晶的時候，<hex:02100103>我就應該強硬地阻止他。
80,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_421,老師是一位非常出色的研究者，<hex:02100103>雖然性格上有點小毛病……
81,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_422,他這個人平時冷靜謙虛，<hex:02100103>但一旦投入研究中，就容易忽略周圍的情況。<hex:02100103>一心沉浸在自己的世界裡，沒少引發誤會。
82,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_423,這次肯定也是這樣，老師專注於搜索水晶<hex:02100103>沒能注意到不對之處，結果發生了意外吧。
83,TEXT_CTSOPN60NEMJIJI_00732_NEMJIJI_000_424,不過我相信老師不會輕易出事，<hex:02100103>他一定可以平安回來。<hex:02100103>話雖如此，只能在這裡等著實在是太煎熬了。
--------------------
這個部分 沒翻譯的留空對吧
--------------------
如果可以的話 煩請大大解一份JP的原始資料壓縮檔供作參考 謝謝

**Comment by GpointChen** [comment]
[rawexd-cut-ja640.zip](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/11562815/rawexd-cut-ja640.zip)

或者你可以用[修改後的SaintCoinach]輸出，但需要先安裝visual studio跟C#。

**Reply by Liteya** [reply]
謝謝 修改後的saint 嘛 我嘗試看看 沒翻譯的部分 在檔案上是直接留白就好了對吧?

**Reply by GpointChen** [reply]
是的。

我剛才又檢查了一遍有類似問題的檔案，已一併修正於 https://github.com/GpointChen/FFXIVChnTextPatch-GP/commit/aaf93ca27aed463e4769ac83498a65823daad1f3 中，如仍有發現問題可再回覆。
